File: app/src/main/java/intellij/class13/deliverableexercise/hash/java/AssistantTeacher.java
package intellij.class13.deliverableexercise.hash.java;

public class AssistantTeacher extends Teacher {
    private Integer quantityMonitoringHours;

    public AssistantTeacher(String name, String lastName, Integer houseTime, Integer teacherCode, Integer quantityMonitoringHours) {
        super(name, lastName, houseTime, teacherCode);
        this.quantityMonitoringHours = quantityMonitoringHours;
    }

    public Integer getQuantityMonitoringHours() {
        return quantityMonitoringHours;
    }

    public void setQuantityMonitoringHours(Integer quantityMonitoringHours) {
        this.quantityMonitoringHours = quantityMonitoringHours;
    }
}

File: src/class13/deliverableexercise/hash/kotlin/AssistantTeacher.kt
package class13.deliverableexercise.hash.kotlin

class AssistantTeacher(
    name: String,
    lastName: String,
    houseTime: Int,
    teacherCode: Int,
    var quantityMonitoringHours: Int
) :
    Teacher(name, lastName, houseTime, teacherCode)